fenix/app/src/main/java/org/mozilla/fenix/settings/search/SaveSearchEngineFragment.kt

```diff
@@ -133,18 +133,28 @@ class SaveSearchEngineFragment : Fragment(R.layout.fragment_save_search_engine)
                 }
 
                 SearchStringValidator.Result.Success -> {
-                    val searchEngine = createSearchEngine(
-                        name,
-                        searchString.toSearchUrl(),
-                        requireComponents.core.icons.loadIcon(IconRequest(searchString))
+                    val update = searchEngine?.copy(
```

Comment:
```suggestion
                    val searchEngine = searchEngine?.copy(
```
Any reason we shouldn't stick with the original variable name? I think we would prefer something more descriptive of what the variable represents.